query/lambda/lambda_function_use_latest_runtime.sql

```diff
@@ -11,6 +11,8 @@ select
   end as reason,
   -- Additional Dimensions
   region,
-  account_id
+  account_id,
+  packagetype
```

Comment:
```suggestion
```

Additional dimensions are typically reserved for the identifying information for a resource, e.g., account ID and region for an AWS regional resource, project and region for a GCP regional resource, in an attempt to standardize output.

I think in this case, it's helpful to show the runtime (which we do in the reason) and we can have a special case for when the `reason` column is `null` like you had in a previous commit.